CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 2.29.0-rc.1 / 2021-08-04
+## 2.29.0-rc.2 / 2021-08-09
```

Comment:
Shouldn't the #9161 fix be added to the rc.2 changelog here?

Reply:
This is part of the kuma_sd, it was not a bug present in the previous release. Since we "reuse" the changelog instead of doing once per RC, I generally only announce them in the release email.

Reply:
What Julien said. I checked previous RCs, that's how we tended to do things for a feature that was only released in this minor.

Reply:
Then we probably need to change our process. Right now the release notes say nothing about the reason why rc.2 was even needed / released. I propose listing all RCs separately in the future in the CHANGELOG (and notes on https://github.com/prometheus/prometheus/releases), until the final release, upon which we sqash them together again.

Reply:
And I think RELEASE.md already says it like I propose: "For release candidates still update `CHANGELOG.md`, but when you cut the final release later, merge all the changes from the pre-releases into the one final update." At least it doesn't say to already squash RCs into each other before the final release.